Ladybird/HelperProcess.cpp

```diff
@@ -25,9 +25,12 @@ ErrorOr<void> spawn_helper_process(StringView process_name, ReadonlySpan<StringV
 
 ErrorOr<Vector<String>> get_paths_for_helper_process(StringView process_name)
 {
+    auto application_path = TRY(ak_string_from_qstring(QCoreApplication::applicationDirPath()));
     Vector<String> paths;
+
     TRY(paths.try_append(TRY(String::formatted("./{}/{}", process_name, process_name))));
-    TRY(paths.try_append(TRY(String::formatted("{}/{}", TRY(ak_string_from_qstring(QCoreApplication::applicationDirPath())), process_name))));
+    TRY(paths.try_append(TRY(String::formatted("{}/{}/{}", application_path, process_name, process_name))));
+    TRY(paths.try_append(TRY(String::formatted("{}/{}", application_path, process_name))));
```

Comment:
I got around to testing this on my machine, and in order to run the WebPlatformTests via web driver, it seems we need it to look like this:

```
   TRY(paths.try_append(TRY(String::formatted("{}/{}/{}", application_path, process_name, process_name))));
    TRY(paths.try_append(TRY(String::formatted("{}/{}", application_path, process_name))));
    TRY(paths.try_append(TRY(String::formatted("{}/../{}", application_path, process_name))));
```
but if I actually review and merge  #17902 then it will only have to be
```
TRY(paths.try_append(TRY(String::formatted("{}/{}", application_path, process_name))));
```
in all cases :thinking: